soc/arm/atmel_sam0/common/soc_saml2x.c

```diff
@@ -0,0 +1,276 @@
+/*
+ * Copyright (c) 2021 Argentum Systems Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Atmel SAML MCU series initialization code
+ */
+
+#include <arch/cpu.h>
+#include <arch/arm/aarch32/cortex_m/cmsis.h>
+#include <device.h>
+#include <init.h>
+#include <kernel.h>
+#include <soc.h>
```

Comment:
please use `<zephyr/>` prefix everywhere

Reply:
Done... `soc.h` remains without the `zephyr/` prefix, correct?

Reply:
correct, soc.h is actually in the process of being removed